_specs/ecip-1000.md

```diff
@@ -73,9 +73,9 @@ A good reason to transfer ownership is because the original author no longer has
 The current ECIP editors are:
 
 * Wei Tang (@sorpaas)
-* Isaac Ardis (@meowsbits)
+* Mr. Meows D. Bits (@meowsbits)
```

Comment:
Please revert those two lines. ECIP-1000 can only be changed when the amendment ECIP is moved to Final.

Reply:
@sorpaas stop maliciously inventing rules that don't exist. This correction has to be done as soon as possible because it is about revealing real names of two participants in the ECIP process, which is a severe violation of pseudonymity, which is, in turn, a severe violation of cryptography and blockchain industry ethics.

Reply:
Okay I completely removed from my mind that soc1c is Afri and meowbits is Isaac. 

Now, answer me -- who are `meowsbits` and `soc1c`? And can you provide justifications why they deserve to be editors? Besides, what are the reasoning behind removing Isaac and Afri as editors?

Reply:
@sorpaas I 100% agree with you, _and i'm sure we are not the only 2_, on how *simply* editing a  final ECIP is against the 'protocol' and even comparable to an "irregular state change"  ( *puff* another unicorn dies* ) or the DAO. It's reverting/editing something that shouldn't be edited without going through the consensus protocol that is currently in place. 

Your second objection to this irregular state change (*puff* ... ) is that it has an hidden goal of trying to cover up for past actions of Afri. Seeing who his doing all this shenanigans, I tend to agree with you but one can never really tell what's in their head.

To that, and to all the **crazy people** here screaming 'doxxing' and tried to remove you from Owner with a shameless 'operation' the past 48h.. 

**The internet does not forget.**
**The internet does not forget.**
**The internet does not forget.**

If Afri and Isaac **REALLY** wish to be pseudo or anonymous, they need to atleast clean up their tracks otherwise it's just silly. They 'doxxed' themselves long time ago. You can't 'undoxx' someone. I'm surprised this is the best excuse you could find to try to kick @sorpaas out of the repo and community. 

**BUT** 

The proposed irregullar state change (*puff*) has zero relevance or impact for ETC..

and most importantly, the ETC values speak higher. 

If an individual asks for a simple thing like this, with zero impact on specifications or anything else, for the sake of **his/hers safety** (even if it does nothing for it or sounds like bullshit) we shouldn't be butting heads too long.

Let's either *kill another unicorn* or push an ammending ECIP as soon as possible, with ONLY this simple or removign their real names from it, instead of wasting more time on this.

But ya'll are crazy for thinking that this will save anyone from harm and this 'doxxing'-gate is plain bullshit coming from **'Team Krykoder'** ... yeah @TokenHash, that's who you are rolling with apparently, you sock puppet. Your lack of focus on the issue and wolfcrying for Team Krykoder will not be forgotten. You accused @sorpaas of personal attacks and other shit when you have ZERO moral ground. Look yourself in the mirror.

Fin.

PS: The Internet Does Not Forget.

Reply:
Slightly off-topic, but indeed, it's kind of interesting to look back at those cover-ups https://web.archive.org/web/20200121172657/https://twitter.com/eth_classic/status/1103720719991607296